src/ui/profile_preferences.ui

```diff
@@ -1341,124 +1341,90 @@
        <string>Mapper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_24">
-       <item row="1" column="0">
-        <widget class="QGroupBox" name="downloadMapOptions">
+       <item row="4" column="0">
```

Comment:
I don't want to be re-ordering the XML files after every ui change. Does not seem like a good use of my time :confused: